Styles/docs.css

```diff
@@ -0,0 +1,130 @@
+.header {
+  z-index: 1;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
```

Comment:
You are using flex in multiple places. So, it will be good if you make utility classes for these and use them everywhere.